.github/workflows/deploy-manual.yml

```diff
@@ -51,3 +51,20 @@ jobs:
         platforms: linux/amd64,linux/arm64,linux/ppc64le,linux/s390x
         push: true
         tags: ${{ steps.tag.outputs.tag }}
+
+    # This change will be staged and committed in the PR pushed below.
+    # The script below will fail if no change was made.
+    - name: update ansible-operator base
+      run: |
+        set -e
+        sed -i -E 's|FROM quay\.io/operator-framework/ansible-operator-base:.+|FROM '"${{ steps.tag.outputs.tag }}"'|' images/ansible-operator/Dockerfile
```

Comment:
is sed happy with those single quotes around the `steps.tag.output.tag`?

Reply:
I'm effectively treating action context value expansion as environment variable expansion, so yeah it should be. Not sure if this is the correct way to do this though, i.e. this might be fine
```sh
sed -i -E 's|FROM quay\.io/operator-framework/ansible-operator-base:.+|FROM ${{ steps.tag.outputs.tag }}|' images/ansible-operator/Dockerfile
```